roles/matrix-bot-maubot/defaults/main.yml

```
@@ -0,0 +1,36 @@
+---
+
+matrix_bot_maubot_enabled: true
+matrix_bot_maubot_container_image_self_build: false
+matrix_bot_maubot_docker_repo: "https://mau.dev/maubot/maubot.git"
+matrix_bot_maubot_docker_src_files_path: "{{ matrix_bot_maubot_base_path }}/docker-src"
+
+matrix_bot_maubot_version: latest
+matrix_bot_maubot_docker_image: "dock.mau.dev/maubot/maubot:{{ matrix_bot_maubot_version }}"
+matrix_bot_maubot_docker_image_force_pull: "{{ matrix_bot_maubot_docker_image.endswith(':latest') }}"
+
+matrix_bot_maubot_base_path: "{{ matrix_base_data_path }}/maubot"
+matrix_bot_maubot_data_path: "{{ matrix_bot_maubot_base_path }}/data"
+matrix_bot_maubot_config_path: "{{ matrix_bot_maubot_base_path }}/config"
+matrix_bot_maubot_container_data_dir: "/data"
+matrix_bot_maubot_container_config_dir: "/root/.config/"
+
+matrix_bot_maubot_bot_server: "https://{{ matrix_server_fqn_matrix }}"
```

Looks like this variable is used in multiple places, both as `public_url` in the configuration and in other places.

I suppose that not all instances need to use this "public URL".

We can define one more variable here:

```suggestion
matrix_bot_maubot_bot_server_url: "{{ matrix_homeserver_container_url }}"
matrix_bot_maubot_bot_server_public: "https://{{ matrix_server_fqn_matrix }}"
```

.. and then use these accordingly.

Not sure if it makes sense though.. Maybe maubot expects all server URLs defined in its configuration to indeed be public? Feel free to disregard this if so.

---

We do need the public url everywhere but for different reasons. I updated accordingly